### PR TITLE
Add tests for territories, refs etalab/udata#399

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

**Comment by davidbgk** [comment]
Tests will fail until https://github.com/etalab/udata/pull/401 is merged.

File: udata_gouvfr/tests.py [hunk 1]
```diff
@@ -248,17 +250,155 @@ class TerritoriesSettings(GouvFrSettings):
...
+        data = response.data.decode('utf-8')
+        self.assertIn('Arles', data)
+        territory_datasets = self.get_context_variable('territory_datasets')
+        self.assertEqual(len(territory_datasets), 9)
+        for dataset in territory_datasets:
+            self.assertIn(
+                '<div data-udata-territory-id="{dataset.slug}"'.format(
+                    dataset=dataset),
+                data)
+        self.assertEqual(
```
Comment (noirbizarre) [inline udata_gouvfr/tests.py]: `assertFalse` ?

File: udata_gouvfr/tests.py [hunk 2]
```diff
@@ -248,17 +250,155 @@ class TerritoriesSettings(GouvFrSettings):
...
+                    dataset=dataset),
+                data)
+        other_datasets = self.get_context_variable('other_datasets')
+        self.assertEqual(len(other_datasets), 3)
+        for dataset in other_datasets:
+            self.assertIn(
+                '<div data-udata-dataset-id="{dataset.id}"'.format(
+                    dataset=dataset),
+                data)
+        self.assertEqual(
```
Comment (noirbizarre) [inline udata_gouvfr/tests.py]: `assertFalse`

File: udata_gouvfr/tests.py [hunk 3]
```diff
@@ -248,17 +250,155 @@ class TerritoriesSettings(GouvFrSettings):
...
+        response = self.client.get(
+            url_for('territories.territory', territory=arles))
+        self.assert200(response)
+        data = response.data.decode('utf-8')
+        territory_datasets = self.get_context_variable('territory_datasets')
+        self.assertEqual(len(territory_datasets), 9)
+        other_datasets = self.get_context_variable('other_datasets')
+        self.assertEqual(len(other_datasets), 3)
+        self.assertEqual(
+            self.get_context_variable('has_pertinent_datasets'), False)
```
Comment (noirbizarre) [inline udata_gouvfr/tests.py]: assertFalse

**Comment by noirbizarre** [comment]
r+wc

**Pull request merged**